packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js

```diff
@@ -0,0 +1,107 @@
+import sinon from 'sinon'
+import React from 'react'
+import TranscriptionReductions from '@store/TranscriptionReductions'
+import taskRegistry from '@plugins/tasks'
+import { TranscribedLines } from './TranscribedLines'
+import { reducedSubject } from '@store/TranscriptionReductions/mocks'
+import { TranscriptionLine } from '@plugins/drawingTools/components'
+
+describe('Component > TranscribedLines', function () {
+  let wrapper, task, consensusLines
+  before(function () {
+    const transcriptionReductions = TranscriptionReductions.create({
+      reductions: [{ data: reducedSubject }],
+      subjectId: '1234',
+      workflowId: '5678'
+    })
+    consensusLines = transcriptionReductions.consensusLines
+    const transcriptionModels = taskRegistry.get('transcription')
+    task = transcriptionModels.TaskModel.create({
+      instruction: 'Transcribe the text',
+      taskKey: 'T1',
+      type: 'transcription'
+    })
+    wrapper = shallow(<TranscribedLines lines={consensusLines} task={task} />)
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
+
+  describe('incomplete lines', function () {
+    let lines
+    before(function () {
+      lines = wrapper.find(TranscriptionLine).find({ state: 'transcribed' })
+    })
+
+    it('should render', function () {
+      const transcribedLines = consensusLines.filter(line => !line.consensusReached)
+      expect(lines).to.have.lengthOf(transcribedLines.length)
+    })
+
+    it('should be focusable', function () {
+      lines.forEach((component) => {
+        const consensusLineWrapper = component.parent()
+        expect(consensusLineWrapper.props().tabIndex).to.equal(0)
+      })
+    })
+
+    xit('should create a new mark on click', function () {
+      
+    })
+
+    it('should have an explanatory tooltip', function () {
+      lines.forEach((component, index) => {
+        const tooltip = wrapper.find({ id: `transcribed-${index}`})
+        expect(tooltip).to.have.lengthOf(1)
+      })
+    })
+  })
+
+  describe('completed lines',function () {
+    let lines, completeLines
+    before(function () {
+      lines = wrapper.find(TranscriptionLine).find({ state: 'complete' })
+      completeLines = consensusLines.filter(line => line.consensusReached)
+    })
+
+    it('should render', function () {
+      expect(lines).to.have.lengthOf(completeLines.length)
+    })
+
+    it('should be labelled with the consensus text', function () {
+      lines.forEach((component, index) => {
+        const consensusLineWrapper = component.parent()
+        expect(consensusLineWrapper.props()['aria-label']).to.equal(completeLines[index].consensusText)
+      })
+    })
+
+    xit('should show the label on hover', function () {
+      lines.forEach((component, index) => {
+        const consensusLineWrapper = wrapper.find({ ['aria-describedby']: `complete-${index}`})
+        consensusLineWrapper.simulate('hover')
+      })
+    })
```

Comment:
Not yet. This test isn't actually passing yet as it has no expectation and the default behavior for a test without an expectation is to pass it, so I marked it as pending so it was clear what the state of this is. I'm not sure how to test this without snapshots because the visibility of the tooltip is being handled by css.